diff --git a/cpp/ycm/CMakeLists.txt b/cpp/ycm/CMakeLists.txt
index e5918d75..adb63549 100644
--- a/cpp/ycm/CMakeLists.txt
+++ b/cpp/ycm/CMakeLists.txt
@@ -70,7 +70,7 @@ if ( USE_CLANG_COMPLETER AND
     set( LIBCLANG_DIRNAME
          "libclang-${CLANG_VERSION}-x86_64-unknown-linux-gnu" )
     set( LIBCLANG_SHA256
-         "23c213ae2dba9cc98d09a4997de60e10f270276335f05f1bacf861678e231809" )
+         "d0a9e10a9b5096c4fcfde591a9de07d7a46a1b22cd422b278459a3482ab76177" )
   else()
     message( FATAL_ERROR
       "No prebuilt Clang ${CLANG_VERSION} binaries for this system. "
@@ -79,11 +79,11 @@ if ( USE_CLANG_COMPLETER AND
       "See the YCM docs for details on how to use a user-compiled libclang." )
   endif()
 
-  set( LIBCLANG_FILENAME "${LIBCLANG_DIRNAME}.tar.bz2" )
+  set( LIBCLANG_FILENAME "clang+llvm-10.0.0-x86_64-unknown-linux-gnu.tar.xz" )
 
   set( LIBCLANG_DOWNLOAD ON )
   set( LIBCLANG_URL
-       "https://dl.bintray.com/ycm-core/libclang/${LIBCLANG_FILENAME}" )
+       "https://github.com/ycm-core/llvm/releases/download/10.0.0/clang+llvm-10.0.0-x86_64-unknown-linux-gnu.tar.xz" )
 
   # Check if the Clang archive is already downloaded and its checksum is
   # correct.  If this is not the case, remove it if needed and download it.
